LayoutTests/imported/w3c/web-platform-tests/svg/animations/seeking-events-5-expected.txt

```diff
@@ -1,3 +1,3 @@
 
-FAIL Seeking forwards does not dispatch 'repeatEvent' assert_true: Not expecting event, but got beginEvent event expected true got false
+FAIL Seeking forwards does not dispatch 'repeatEvent' assert_true: Not expecting event, but got repeatEvent event expected true got false
```

Comment:
Does this mean we are dispatching events but we should not be? Is that something we can do right when adding `repeatEvent`, or does it need to be fixed separately?

Reply:
Yes - we need to fix the cases for dispatch events and in separate. We still don't expose 'onbegin' and others through IDL interface.